mantle/kola/harness.go

```diff
@@ -272,6 +273,73 @@ func testRequiresInternet(test *register.Test) bool {
 	return false
 }
 
+type DenyListObj struct {
+	Pattern string   `yaml:"pattern"`
+	Tracker string   `yaml:"tracker"`
+	Streams []string `yaml:"streams"`
+	Arches  []string `yaml:"arches"`
+}
+
+type ManifestData struct {
+	AddCommitMetadata struct {
+		FcosStream string `yaml:"fedora-coreos.stream"`
+	} `yaml:"add-commit-metadata"`
+}
+
+func parseDenyListYaml() {
+	var objs []DenyListObj
+
+	// Parse kola-denylist into structs
+	denyListFile, err := ioutil.ReadFile("src/config/kola-denylist.yaml")
+
+	if err != nil {
+		return
+	}
+
+	plog.Debug("Found kola-denylist.yaml")
+	err = yaml.Unmarshal(denyListFile, &objs)
+
+	if err != nil {
+		return
+	}
+
+	plog.Debug("Parsed kola-denylist.yaml")
+
+	// Get stream and arch
+	stream := ""
+
+	manifestFile, err := ioutil.ReadFile("src/config/manifest.yaml")
+	var manifest ManifestData
+
+	if err == nil {
+		plog.Debug("Found manifest.yaml")
+		err = yaml.Unmarshal(manifestFile, &manifest)
+	}
+
+	if err == nil {
+		plog.Debug("Parsed manifest.yaml")
+		stream = manifest.AddCommitMetadata.FcosStream
+	}
```

Comment:
Need to handle the `err != nil` case. Maybe something like:

```suggestion
    if err != nil {
        plog.Fatal(err)
    }
    err = yaml.Unmarshal(manifestFile, &manifest)
    if err != nil {
        plog.Fatal(err)
    }
    stream = manifest.AddCommitMetadata.FcosStream
```

Reply:
Got it. `err == nil` is a little awkward to read aswell; your change is more clean imo.